src/server/pfs/cmds/cmds.go

```diff
@@ -739,6 +739,32 @@ files into your Pachyderm cluster.
 		}),
 	}
 
```

Comment:
I think it might be good to have them there even though they're lower level. They came in handy for debugging stuff and I could definitely see wanting to have a user debug something with their `pachctl`, that's the only use case I can think of though.